src/styles/index.scss

```diff
@@ -1,3 +1,66 @@
-body {
-  margin: 0;
+$gap-size: 1px;
+$cells-in-row: 7;
+$height-cell: 100px;
+$width-cell: 100px;
+$week-days:
+  'mon' 1,
+  'tue' 2,
+  'wed' 3,
+  'thu' 4,
+  'fri' 5,
+  'sat' 6,
+  'sun' 7;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $width-cell * $cells-in-row + $gap-size * ($cells-in-row - 1);
+  gap: $gap-size;
+
+  &__day {
+    height: $height-cell;
+    width: $width-cell;
+    background-color: #eee;
```

Comment:
The color value `#eee` should be stored in a variable for better maintainability and to follow the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).